Resources/Textures/_CP14/Structures/Wallmount/wallmount_painting.rsi/meta.json

```
@@ -0,0 +1,27 @@
+{
+  "version": 1,
+  "license": "CLA",
+  "copyright": "Created by ripmax for CrystallPunk14",
```

```suggestion
  "copyright": "Created by Max Gab for CrystallPunk14",
```
он так попросил себя указывать